src/style.css

```
@@ -0,0 +1,140 @@
+@font-face {
+  font-family: Avenir_heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: avenir_book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  font-weight: 300;
+}
+
+.dandruff--small {
+  block-size: 11px;
+}
+
+.dandruff {
+  display: flex;
+  align-items: center;
+}
+
+.search {
+  border: none;
+  outline: none;
+  margin: 17px;
+  display: flex;
+  width: 158px;
+  height: 19px;
+  padding-right: 0;
+  align-items: center;
+  flex: 1;
+}
+
+.search--small {
+  border: none;
+  outline: none;
+  margin: 9px;
+  display: flex;
+  width: 158px;
+  height: 19px;
+  padding-right: 0;
+  align-items: center;
+  flex: 1;
+}
+
+.search::placeholder,
+.search--small::placeholder {
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+}
+
+.search::placeholder {
+  font-size: 16px;
+}
+
+.search--small::placeholder {
+  font-size: 14px;
+}
+
+.form {
+  margin-top: 20px;
+  height: 70px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding-left: 26px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.form--small {
+  margin-top: 20px;
+  height: 42px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding-left: 13px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.form--small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
```

No need to repeat it twice. Just add the `form` class to the small form as well. The same goes for the code below
```suggestion
.form:hover {
  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
}
```